### PR TITLE
Admin Dashboard: Show box Devices and Devices over Time

### LINKED ISSUE #2706: Admin Dashboard: Show box "Devices" and "Devices over Time"
In the admin dashboard, please add:
1. A box "Devices" with info:
* Number of All devices
* Number of Connected Devices
* Number of Inactive Devices
* Number of Deleted Devices
* Number of Users that have Devices Connected (users can connect several devices, so this number is relevant as well)
2. A chart with Connected Devices over Time
Screenshot from 2019-07-29 08-46-52

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2706 

Changes: 

Demo Link: https://pr-2752-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]
Screenshot 2019-08-03 at 10 40 13 AM
Screenshot 2019-08-03 at 10 40 20 AM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@akshatnitd API `getUsers.json?getDeviceStats=true` gives internal server error was working well on the local server. https://github.com/fossasia/susi_server/pull/1351

File: src/components/Admin/AdminTab/AdminTab.js [hunk 1]
```diff
@@ -184,11 +204,14 @@ class AdminTab extends React.Component {
...
-    const dateSortedData = sortByDate(data);
-    // The dateSortedData array has multiple entries for same date, add the count of same dates
-    const mergedateSortedData = mergeByDate(dateSortedData);
-    const chartData = mergedateSortedData.map(data => {
+  renderChart = ({ data, heading, legend, sort = true }) => {
+    let DataOverTime = data;
+    if (sort) {
+      const dateSortedData = sortByDate(data);
+      // The dateSortedData array has multiple entries for same date, add the count of same dates
+      DataOverTime = mergeByDate(dateSortedData);
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: dataOverTime

File: src/components/Admin/AdminTab/AdminTab.js [hunk 2]
```diff
@@ -345,6 +369,20 @@ class AdminTab extends React.Component {
                   </CardContentContainer>
                 </Card>
               </CardContainer>
+              <CardContainer>
+                <Card>
+                  <CardHeading>Devices</CardHeading>
+                  <Divider />
+                  <CardContentContainer>
+                    <Typography variant="body1" gutterBottom>
+                      Connected: {connectedDevices}
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: Connected devices:

File: src/components/Admin/AdminTab/AdminTab.js [hunk 3]
```diff
@@ -345,6 +369,20 @@ class AdminTab extends React.Component {
...
+                    </Typography>
+                    <Typography variant="body1" gutterBottom>
+                      Users: {deviceUsers}
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: User with devices :

**Comment by harsh-jindal** [comment]
@akshatnitd Done

**Comment by harsh-jindal** [comment]
@fragm3 Done

**Pull request merged**